src/Navigation.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { Navbar, Nav } from "react-bootstrap";
+import { Link, withRouter } from "react-router-dom"
+import Cookies from "js-cookie";
+import _ from "underscore";
+
+
+function Navigation() {
+    const user = Cookies.get("user");
+    console.log(user);
+    return (
+        <div>
+            <Navbar bg="light" expand="lg">
+                <Navbar.Brand href="/">
+                    <img
+                        className="d-inline-block align-top"
+                        src="public/assets/images/AnitaBLogo.png"
+                        alt="Logo"
+                    />
+                </Navbar.Brand>
+                <Navbar.Toggle aria-controls="basic-navbar-nav" />
+                <Navbar.Collapse id="basic-navbar-nav">
+                    <Nav className="mr-auto">
+                        <Nav.Link tag={Link} href="/">Home</Nav.Link>
+                        <Nav.Link tag={Link} href="/members">Members</Nav.Link>
+                        <Nav.Link tag={Link} href="/my-space">My Space</Nav.Link>
+                        {_.isEmpty(user) &&
+                            <Nav>
+                                <Nav.Link tag={Link} href="/register">Register</Nav.Link>
```

Comment:
you need to check it twice. can just use conditional rendering.

_.isEmpty(user) ? <component /> : <other component/>

Reply:
Done that before in the previous commit. just want to experiment on different types of if conditions on javascript for fun 😁. I'll revert it back 👍

Reply:
update. now this is reverted back to conditional rendering 😁